lib/matplotlib/_layoutbox.py

```diff
@@ -623,29 +623,21 @@ def match_margins(boxes, levels=1):
 
 
 def seq_id():
-    '''
-    Generate a short sequential id for layoutbox objects...
-    '''
-
-    global _layoutboxobjnum
```

Comment:
Why has this lost the `global`?

Reply:
you don't need `global` to read from a global (... just like you can call a module-level function from another function without declaring it as global).